### PR TITLE
WEBEU2 - User Interface - Rui Hildt 🇧🇪

### PR DESCRIPTION
**Pull request by ruihildt**
This is a test of the git flow.

### CODE REVIEW

File: great-idea-website/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
 
   <title>Great Idea!</title>
-
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
```
Comment (maximesalomon) [inline great-idea-website/index.html]: Google fonts ✌️

File: great-idea-website/index.html [hunk 2]
```diff
@@ -14,50 +14,87 @@
...
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </nav>
+      
+    </header>
+
+    <div class="main-content">
```
Comment (maximesalomon) [inline great-idea-website/index.html]: There is a `<main>` HTML5 tag now :)

File: great-idea-website/index.html [hunk 3]
```diff
@@ -14,50 +14,87 @@
...
+
+      <section class="row-1">
+        <div class="column">
+          <h1>Innovation On Demand</h1>
```
Comment (maximesalomon) [inline great-idea-website/index.html]: To be pixel perfect here you might want to use `</br>`

File: great-idea-website/css/index.css [hunk 4]
```diff
@@ -63,4 +64,95 @@ h1, h2, h3, h4, h5 {
...
\ No newline at end of file
+/* Your code starts here! */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+}
+
+a, p {
+	font-family: roboto;
```
Comment (maximesalomon) [inline great-idea-website/css/index.css]: 🤖

File: great-idea-website/css/index.css [hunk 5]
```diff
@@ -63,4 +64,95 @@ h1, h2, h3, h4, h5 {
...
+}
+
+
+a {
+	text-decoration:none;
+	color:grey;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/index.css]: Might want to add some hover here :)

File: great-idea-website/css/index.css [hunk 6]
```diff
@@ -63,4 +64,95 @@ h1, h2, h3, h4, h5 {
...
+
+h1 {
+	font-size:4.2em;
```
Comment (maximesalomon) [inline great-idea-website/css/index.css]: Great use of em

File: great-idea-website/css/index.css [hunk 7]
```diff
@@ -63,4 +64,95 @@ h1, h2, h3, h4, h5 {
...
+	background-color: #c0caeb;
+	padding: 20px 5px;
+}
+nav .first-anchor {
+  margin-left: 10px;
+}
+
+nav a, nav img{
+	display:inline-block;
+	vertical-align: middle;
```
Comment (maximesalomon) [inline great-idea-website/css/index.css]: Pixel perfect eyes 👀

File: great-idea-website/index.html (hunk 3 above)
Comment (ruihildt) [reply great-idea-website/index.html]: I don't use </br>, because of responsive issues. (largely occurring with larger text, it's true)

**Pull request merged**